packages/wrangler/src/user.tsx

```diff
@@ -961,13 +974,11 @@ export async function logout(): Promise<void> {
   });
   await response.text(); // blank text? would be nice if it was something meaningful
   console.log(
-    "💁  Wrangler is configured with an OAuth token. The token has been successfully revoked"
+    "You were authenticated via OAuth, which has been successfully revoked."
```

Comment:
"which" doesn't refer to a noun here, so it makes this sentence sound weird. I say we don't even log this message.